public/resources/fontello-24c5e6ad/demo.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
```

Comment:
**Proposal:** Let's remove the following files:

- public/resources/fontello-24c5e6ad/LICENSE.txt
- public/resources/fontello-24c5e6ad/README.txt
- public/resources/fontello-24c5e6ad/demo.html

Then let's copy the contents of public/resources/fontello-24c5e6ad/LICENSE.txt into our root README.md in the **Licenses** section below our MPL-2.0 notice.